clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -906,6 +906,7 @@ public long position(TopicPartition partition, Duration timeout) {
                     return position.offset;
 
                 updateFetchPositions(timer);
```

Comment:
We need to make sure `updateFetchPositions` also honour the `wakeup`. For example, the following test get failed with this PR.

```java
  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
  @Timeout(5)
  def testPositionRespectsWakeup(quorum: String, groupProtocol: String): Unit = {
    val topicPartition = new TopicPartition(topic, 15)
    val properties = new Properties();
    properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:12345");
    val consumer = createConsumer(configOverrides = properties)
    consumer.assign(List(topicPartition).asJava)

    CompletableFuture.runAsync { () =>
      TimeUnit.SECONDS.sleep(1)
      consumer.wakeup()
    }

    assertThrows(classOf[WakeupException], () => consumer.position(topicPartition, Duration.ofSeconds(100)))
  }
```

Reply:
`properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:12345");` can cause the connection waiting, and that should be "wakeupable"

Reply:
I add `FetchCommittedOffsetsEvent` to `wakeupTrigger.setActiveTask`. Thanks for the review.